scripts/pages/leaderboards/leaderboards.ts

```diff
@@ -159,7 +159,8 @@ class LeaderboardsHandler {
 	onMapLeaderboardsLoad(map: MMap) {
 		this.panels.tracksDropdown.RemoveAllOptions();
 
-		const currentMode = GameModeAPI.GetCurrentGameMode();
+		const isTabMenu = $.GetContextPanel().id === 'TabMenuLeaderboard';
```

Comment:
`this.panels.cp` instead of `$.GetContextPanel` - JS object lookup is generally faster than calling into C++. Also I would pull this out to a readonly member variable.